data.sql

```diff
@@ -9,5 +9,18 @@ INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) V
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (7, 'Squirtle', '1993-04-02', 3, FALSE, -12.13);
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (8, 'Angemon', '2005-06-12', 1, TRUE, -45);
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (9, 'Boarmon', '2005-06-07', 7, TRUE, 20.4);
-INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (10, 'Blossom', '1998-10-13', 3, TRUE, 17); 
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (10, 'Blossom', '1998-10-13', 3, TRUE, 17);
 
+INSERT INTO owners (full_name,age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melody', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+
+UPDATE animals SET species_id=1;
+UPDATE animals SET species_id=2 WHERE name LIKE '%mon';
+
+UPDATE animals SET owner_id=1 WHERE name='Agumon';
```

Comment:
[Optional]
- [ ] You should make a query with the name instead of setting the id. 
On actual-world projects, there is no way to know all ids for all records.

For Example : 
```sql
INSERT INTO specializations (species_id, vet_id) VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));


INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');
```